packages/scandipwa/src/component/CheckoutPayments/CheckoutPayments.component.js

```diff
@@ -114,6 +117,12 @@ export class CheckoutPayments extends PureComponent {
 
     renderPayments() {
         const { paymentMethods } = this.props;
+        const { totals: { grand_total } } = this.props;
+        const paymentFree = paymentMethods.find((payment) => payment.code === 'free');
```

Comment:
Please don't use such strings as is.
Either put it in a constant or reuse it from the existing constant.
Putting machine names in the constants instead of using strings can prevent issues like making a typo in a string - if you make a typo in the constant name, js will fire the error.